src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -120,10 +120,42 @@ private boolean setSystemPropertyOrClientSetting(
   }
 
   private static boolean handleGameSetting(final String key, final String value) {
-    if (GameRunner.MAP_FOLDER.equals(key)) {
+    if (CliProperties.MAP_FOLDER.equals(key)) {
       ClientSetting.MAP_FOLDER_OVERRIDE.save(value);
       return true;
     }
     return false;
   }
+
+
+  /**
+   * An interface for all CLI related constants.
+   */
+  public interface CliProperties {
+    // argument options below:
+    static final String TRIPLEA_GAME_HOST_CONSOLE_PROPERTY = "triplea.game.host.console";
+    static final String TRIPLEA_GAME_PROPERTY = "triplea.game";
+    final String TRIPLEA_MAP_DOWNLOAD_PROPERTY = "triplea.map.download";
+    static final String TRIPLEA_SERVER_PROPERTY = "triplea.server";
+    static final String TRIPLEA_CLIENT_PROPERTY = "triplea.client";
+    static final String TRIPLEA_HOST_PROPERTY = "triplea.host";
+    static final String TRIPLEA_PORT_PROPERTY = "triplea.port";
+    static final String TRIPLEA_NAME_PROPERTY = "triplea.name";
+    static final String TRIPLEA_SERVER_PASSWORD_PROPERTY = "triplea.server.password";
+    static final String TRIPLEA_STARTED = "triplea.started";
+    static final String LOBBY_HOST = "triplea.lobby.host";
+    static final String LOBBY_GAME_COMMENTS = "triplea.lobby.game.comments";
+    static final String LOBBY_GAME_HOSTED_BY = "triplea.lobby.game.hostedBy";
+    static final String LOBBY_GAME_SUPPORT_EMAIL = "triplea.lobby.game.supportEmail";
+    static final String LOBBY_GAME_SUPPORT_PASSWORD = "triplea.lobby.game.supportPassword";
+    static final String LOBBY_GAME_RECONNECTION = "triplea.lobby.game.reconnection";
+    static final String TRIPLEA_ENGINE_VERSION_BIN = "triplea.engine.version.bin";
```

Comment:
The `TRIPLEA_` prefix on a subset of these constants seems redundant.  We should consider removing it to keep the naming consistent.  I'm good if you want to punt that to a follow-up PR to keep the diff here straightforward.